lightning/src/routing/network_graph.rs

```diff
@@ -1579,68 +1546,25 @@ mod tests {
 
 		let node_1_privkey = &SecretKey::from_slice(&[42; 32]).unwrap();
 		let node_2_privkey = &SecretKey::from_slice(&[41; 32]).unwrap();
-		let node_id_1 = PublicKey::from_secret_key(&secp_ctx, node_1_privkey);
-		let node_id_2 = PublicKey::from_secret_key(&secp_ctx, node_2_privkey);
-		let node_1_btckey = &SecretKey::from_slice(&[40; 32]).unwrap();
-		let node_2_btckey = &SecretKey::from_slice(&[39; 32]).unwrap();
 
-		let zero_hash = Sha256dHash::hash(&[0; 32]);
-		let short_channel_id = 0;
-		let chain_hash = genesis_block(Network::Testnet).header.block_hash();
 		let amount_sats = 1000_000;
+		let short_channel_id;
 
 		{
```

Comment:
I think some of the scopes in these tests are hanging around from a former locking mechanism. Probably could remove them if you're touching most of the code within them. Would simplify the `short_channel_id` assignment here and elsewhere, for instance.

Reply:
Yea, would prefer to just do this later. Like Val noted above there's more to clean up here, but changing all the tests is pretty unrelated to this PR so I'm hesitant to Do It All right now.